### ISSUE TITLE
Refactor Cassandra page

### ISSUE THREAD

**Issue by pvalena** [developer-portal/content#282]
See https://github.com/developer-portal/content/pull/260.

**Comment by tpardeshi** [comment]
@pvalena I would like to contribute to this issue. It would be helpful if you become my mentor for my first contribution to fedora. Please let me know how can I help.

**Comment by pvalena** [comment]
Hello @tpardeshi, are you familiar with Markdown and/or Cassandra usage?

Please also check our [Contributing guide](https://developer-portal.github.io/contributing/). I'm available on IRC if you have more questions.

**Comment by tpardeshi** [comment]
@pvalena Yes I am familiar with Cassandra. Not so hands-on but yes I can contribute to documentation.

**Comment by pvalena** [comment]
> @pvalena Yes I am familiar with Cassandra. Not so hands-on but yes I can contribute to documentation.

Have you checked the documentation. Have you found it straghtforward? You can also check other pages (and their source) to know where to begin.

If you're not familiar with git, and working on commandline, it's completely fine to simply create the page using github editing tools. Please create a Pull Request, and ping me on irc in case you have some questions. When you create the PR, I'll be able to see the changes, advise, proprose enhancements etc.. And I can also make some testing builds, possibly.

**Comment by tpardeshi** [comment]
Hi @pvalena Sorry for late response. I have created a [PR](https://github.com/developer-portal/content/pull/342) for this issue. Please review. Thanks.
